move-vm-backend-common/tests/assets/move-projects/signer_scripts/sources/Generics.move

```diff
@@ -0,0 +1,48 @@
+script {
```

Comment:
There could be a test with generic function but that generic shouldn't be present in the parameters list.
That test shouldn't fail.

Reply:
Done